packages/SketchMorph2-Core.package/M2Backend.class/instance/importImage..st

```diff
@@ -0,0 +1,10 @@
+tool selection
+importImage: aFileName
+	| tempForm |
+	self switchMode.
+	tempForm := (Form fromFileNamed: aFileName).
+	self form setExtent: (tempForm extent) depth: self defaultDepth.
```

Comment:
Why are we now using the `self defaultDepth` instead of the `tempForm depth`?

Reply:
To avoid weird artifacts of depth mismatches between the imported image and our canvas.

Reply:
Ah, I see. Fine then.